### PR TITLE
tests/nested/core/core20-create-recovery: verify that recovery system can be created at runtime

### PR DESCRIPTION
**Pull request by bboozzoo**
~~Based on #10254~~

### CODE REVIEW

File: daemon/api_debug.go [hunk 1]
```diff
@@ -329,6 +343,8 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		}
 		st.Prune(opTime, 0, 0, 0)
 		return SyncResponse(true, nil)
+	case "create-recovery":
+		return createRecovery(st, a.Params.RecoverySystemLabel)
```
Comment (stolowski) [inline daemon/api_debug.go:347]: Shouldn't this be restricted to root only?

Comment (bboozzoo) [reply daemon/api_debug.go:347]: This is a handler for POST, for which *OK flags do not apply.

**Review by stolowski — APPROVED** [review]
LGTM

**Review by mvo5 — COMMENTED** [review]
This looks very nice, thank you! I'm fine with having this a debug API initially but we also have a task in our list to create a proper API for this. I have no strong opinion if we should land this first and then move to the real API later or if we go with the real API directly. I linked from "SNAP-11" in jira to this PR fwiw.

**Review by mvo5 — APPROVED** [review]

File: daemon/api_debug.go (hunk 1 above)
Comment (pedronis) [reply daemon/api_debug.go:347]: this should be fine once  https://github.com/snapcore/snapd/pull/10311 lands

**Review by pedronis — APPROVED** [review]
this should be alright once #10311 lands

**Comment by bboozzoo** [comment]
The failures are unrelated, @pedronis @mvo5 can you land it?

**Pull request merged**